### DISCUSSION TITLE
AcidISlands Module - Enabled Scheamtics

### DISCUSSION THREAD

**Discussion by Zsomi** [BG-Software-LLC/SuperiorSkyblock2#1740] [Pending Feature Requests] [answered]
You need a solution for SSB-AcidIslands that only puts the water on the schematic island, not on all islands. I would like to create it as a separate island.

The config would have an empty section where you could list the enabled schematics.

**Comment by Lemvs** [comment, marked as answer]
AcidIslands module replace original world generator with water generator, islands are still in the same world, so this option is not possible, it doesn't work the same as in OneBlock module.

@OmerBenGera you can mark this answered and close it.
